ARCHITECTURE.md

```diff
@@ -36,7 +36,9 @@ SkyPortal uses a PostgreSQL database to manage persistent state. The SkyPortal P
 Each database table is represented by a Python class, and each table Column
 is represented by a class attribute.
 
-The models defined in SkyPortal can be found in the directory `skyportal/models`, while those found in baselayer can be found in `baselayer/app/models.py`. When adding a new model, one of the GitHub actions automatically will create the alembic migration file, which should then be committed in the directory `alembic/versions`.
+The models defined in SkyPortal can be found in the directory `skyportal/models`, while those found in baselayer can be found in `baselayer/app/models.py`.
+
+When adding a new model, database migrations will be key. In addition to the Database Migrations section, which describes how to create migration files locally, one of the GitHub actions automatically will create the alembic migration file, which should then be committed in the directory `alembic/versions`.
```

Comment:
`In addition to the Database Migrations section` -> `In addition to the Database Migrations section of the documentation` ?